src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -529,7 +529,6 @@ def _run(
         _XdsUpdateClientConfigureServicer(channel_configs, args.qps),
         _global_server,
     )
-    channelz.add_channelz_servicer(_global_server)
```

Comment:
Remove this line because we're also doing it in `grpc_admin.add_admin_servicers` below.